packages/common/src/utils/fetcher.ts

```diff
@@ -1,6 +1,6 @@
-import { IFetcher } from './interfaces';
-import { handlePromise } from '../handle-promise/handle-promise';
-import { isZodError } from '../is-zod-error/is-zod-error';
+import { IFetcher } from '../../../../apps/backoffice-v2/src/utils/fetcher/interfaces';
```

Comment:
In monorepo context we shouldn't reference explicitly to resources by the folder structure, and use monorepo ability to reference local packages